src/collections/blog/2019-07-09-Meshery-start-feat-WSL2-K3d/index.mdx

```diff
@@ -18,7 +18,7 @@ redirect_from:
  - /blog/2019-07-09-getting-started-with-Meshery-WSL2-k3d/
 ---
 
-import { BlogWrapper } from "../Blog.style.js";
+import { BlogWrapper } from '../Blog.style.js';
```

Comment:
Considering we have the `quotes: ["error", "double"]` rule defined. I wonder how come the linter didn't throw an error on this change 🤔

Reply:
Oh, double-quotes. Ok, got it.